lib/v2/oreno3d/radar.js

```diff
@@ -1,51 +1,36 @@
 module.exports = {
     'oreno3d.com': {
-        _name: '俺の3Dエロ動画',
+        _name: 'oreno3d',
         '.': [
             {
                 title: '关键词搜索',
                 docs: 'https://docs.rsshub.app/anime.htm#an-の3dエロ-dong-hua-oreno3d-guan-jian-ci-sou-suo',
-                source: ['/search'],
-                target: (_params, url) => {
-                    const searchParams = new URL(url).searchParams;
-                    return `/oreno3d/search/${searchParams.get('keyword')}${searchParams.has('sort') ? '/' + searchParams.get('sort') : ''}`;
-                },
+                source: '/search?sort=:sort?&keyword=:keyword&page=pagelimit:',
```

Comment:
You need to use [WHATWG URL API](https://nodejs.org/api/url.html) to match search parameters

Reply:
I've used it to match the pages which users want

Reply:
`source` can [only match URL path](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze-source). You need to use target function with [WHATWG URL API](https://nodejs.org/api/url.html) to match search parameters

Reply:
I've used it to match the pages in `radar.js`